wide.md

```diff
@@ -0,0 +1,82 @@
+# The ODM Wide Table
+
+The multiple tables of the data model make it easy to store various types of data. Though the ODM structure allows for flexibility and specificity when committing data, this structure is not optimal for consulting and displaying the data, especially when trying to follow the evolution of different quantities over time. To serve this purpose, we propose a systematic template to generate a table that distributes the various recorded time series from a given site into separate columns. The data is spread across rows according to time. We call this table, the "wide" table, since it spreads data horizontally into individual columns.
```

Comment:
"The 'wide' table has one row for each day; whereas, the regular ODM tables have one row for each measurement."